tf2_eigen/test/tf2_eigen-test.cpp

```diff
@@ -28,13 +28,9 @@
 
 
 #include <tf2_eigen/tf2_eigen.h>
-#include <ros/ros.h>
 #include <gtest/gtest.h>
 #include <tf2/convert.h>
 
-static const double EPS = 1e-3;
-
-
 TEST(TfEigen, ConvertVector)
```

Comment:
Both of these fixes can go upstream to ROS1 tf2_eigen.  I'll open a PR for these.